index.html

```diff
@@ -101,6 +102,36 @@ <h2>Connect</h2>
   </section>
 
   <img src ="DGM.jpg" alt="A photo of the website creator.">
+
+  <section id="messageform">
+    <h2>Leave a Message!</h2>
+
+    <form name="leave_message" class="message-form">
+      <br>
+      <label for="usersNameId">Name:</label>
```

Comment:
nit: if your purpose is to connect the label to the form, then you need to have the corresponding ID in the `input` element.

in this label, `usersNameId` is the `for` attribute. So the `input` element must have an `id` of `usersNameId`

same for all other label/input pairs